Library/Formula/nim.rb

```diff
@@ -20,8 +21,32 @@ def install
     end
     system "/bin/sh", "install.sh", prefix
 
-    bin.install_symlink prefix/"nim/bin/nim"
-    bin.install_symlink prefix/"nim/bin/nim" => "nimrod"
+    # Nim doesn't look for a config file in /usr/local, hence we need to make
+    # sure to always pass the exact location of the path
+
+    wrapping_script = bin + "nim"
+
+    wrapping_script.write <<-EOS.undent
+      #!/bin/bash
+
+      # Actual path of the nim compiler
+      nim="#{prefix}/nim/bin/nim"
+
+      if [ -z "$1" ]; then
+        # There were no args
+        exec "$nim"
+      else
+        # Take the first arg, the command and then append the path
+        command="$1"
+        shift
+        exec "$nim" $command --path:"#{prefix}/nim/lib" $@
```

Comment:
Should `$command` and `$@` be quoted here, too? e.g. `exec "$nim" "$command" --path:"#{prefix}/nim/lib" "$@"`